### PR TITLE
acrn-config: supply optional passthrough device for vm

### PR DESCRIPTION
**Pull request by liuwei987**
To match launch config xml, supply more optional passthrough device to
vm.

Tracked-On: #3812
Signed-off-by: Wei Liu <weix.w.liu@intel.com>
Acked-by: Victor Sun <victor.sun@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: misc/acrn-config/launch_config/pt.py [hunk 1]
```diff
@@ -215,7 +215,7 @@ def gen_pt(names, sel, vmid, config):
 def gen_pt_head(names, sel, vmid, config):
 
     # get passthrough device for specify board and uos
-    cap_pt = launch_cfg_lib.get_board_pt_dev(names, vmid)
+    cap_pt = launch_cfg_lib.get_pt_dev()
```
Comment (binbinwu1) [inline misc/acrn-config/launch_config/pt.py]: The modification unify the list for different boards/UOS? What if user choose a device that doesn't exist on the using board?

Comment (liuwei987) [reply misc/acrn-config/launch_config/pt.py]: The available pass through device which show upon on the webUI are from the board info.
if there is no pass through device for PT, then webUI will show empty which user have not chance to chose.

Comment (binbinwu1) [reply misc/acrn-config/launch_config/pt.py]: OK, thanks for explanation. However, the comment "_get passthrough device for specify board and uos_" is still valid after the change here?

Comment (liuwei987) [reply misc/acrn-config/launch_config/pt.py]: > The available pass through device which show upon on the webUI are from the board info.
> if there is no pass through device for PT, then webUI will show empty which user have not chance to chose.

At the meantime, user will know there is no device on the board if webUI show the empty of the pass through device.

Comment (liuwei987) [reply misc/acrn-config/launch_config/pt.py]: > OK, thanks for explanation. However, the comment "_get passthrough device for specify board and uos_" is still valid after the change here?

the comment already remove in the PR.

Comment (liuwei987) [reply misc/acrn-config/launch_config/pt.py]: > OK, thanks for explanation. However, the comment "_get passthrough device for specify board and uos_" is still valid after the change here?

i will update the PR, without the comment.

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Pull request merged**